resources/dmmd-preview.js

```diff
@@ -1,10 +1,20 @@
 $(function () {
     window.register_dmmd_preview = function ($preview) {
+
+        var $form = $preview.parents('form').first();
         var $update = $preview.find('.dmmd-preview-update');
         var $content = $preview.find('.dmmd-preview-content');
         var preview_url = $preview.attr('data-preview-url');
         var $textarea = $('#' + $preview.attr('data-textarea-id'));
 
+        // submit the form if ctrl + enter is pressed in pagedown textarea
+        $textarea.keydown(function (ev) {
+            // Ctrl-Enter pressed
```

Comment:
Be consistent in comments, used "ctrl + enter" above and "Ctrl-Enter" here, "Ctrl+Enter" everywhere sounds good.